test/constexpr_icmp.ll

```diff
@@ -0,0 +1,48 @@
+; RUN: llvm-as -opaque-pointers=0 %s -o %t.bc
+; RUN: llvm-spirv %t.bc  -spirv-ext=+SPV_INTEL_vector_compute -opaque-pointers=0 -o %t.spv
```

Comment:
```suggestion
; RUN: llvm-as %s -o %t.bc
; RUN: llvm-spirv %t.bc  -spirv-ext=+SPV_INTEL_vector_compute -o %t.spv
```

btw, why do we need SPV_INTEL_vector_compute here?

Reply:
I think the failure:

error: line 29: ID '32[%32]' has not been defined

is because llvm.dbg.value is declared and not defined.   Other tests that declare llvm.dbg.value (complex-constexpr-vector and lshr-constexpr.ll) do not validate with spirv-val.

If the option SPV_INTEL_vector_compute is left out, there is an assertion failure:

llvm-spirv: /iusers/lujohn/src/SPIRV-Team/SPIRV-LLVM-Translator/lib/SPIRV/libSPIRV/SPIRVType.h:334: virtual void SPIRV::SPIRVTypeVector::validate() const: Assertion `CompCount == 2 || CompCount == 3 || CompCount == 4 || CompCount == 8 || CompCount == 16' failed.